src/main/resources/application.properties

```diff
@@ -11,3 +11,5 @@ server.port=3007
 server.tomcat.max-threads=5
 codehub.admin.api.configurations.index=configurations
 codehub.admin.api.configurations.default=codehub-default-configuration
+codehub.admin.api.configurations.images.list=http://d66sfbdx0p5w.cloudfront.net/images-category.json
```

Comment:
I think hardcoding this is fine for now and this route is public so no real need to hide it - but eventually we might want to generify this more somehow. Looks good for now though.